docs/releases/v3.0.2.md

```diff
@@ -0,0 +1,33 @@
+# Logging Core Module Release 3.0.2
+
+Welcome to the latest release of `logging` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution)
+maintained by team SIGHUP.
```

Comment:
GitHub respects the newlines in releases notes (it does not join the lines like in all the other places 🤷 )

```suggestion
Welcome to the latest release of `logging` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by team SIGHUP.
```